### PR TITLE
added page 22

### PR DESCRIPTION
**Pull request by EvilShadow1518**
added page 22 and created its button

### CODE REVIEW

File: lib/pages/page_22.dart [hunk 1]
```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+
+class Page22 extends StatelessWidget {
```
Comment (IIdroyII) [inline lib/pages/page_22.dart:3]: It will be Page23

File: lib/home_page.dart [hunk 2]
```diff
@@ -20,6 +20,7 @@ import 'pages/page_18.dart';
 import 'pages/page_19/page_19.dart';
 import 'pages/page20.dart';
 import 'pages/page_21.dart';
+import 'pages/page_22.dart';
```
Comment (IIdroyII) [inline lib/home_page.dart:23]: page_23.dart

File: lib/home_page.dart [hunk 3]
```diff
@@ -54,6 +55,7 @@ class HomePage extends StatelessWidget {
           PageButton(pageNumber: 19, page: Page19()),
           PageButton(pageNumber: 20, page: Page20()),
           PageButton(pageNumber: 21, page: Page21()),
+          PageButton(pageNumber: 22, page: Page22()),
```
Comment (IIdroyII) [inline lib/home_page.dart:58]: PageButton(pageNumber: 23, page: Page23()),

**Review by IIdroyII — CHANGES_REQUESTED** [review]

File: lib/pages/page_22.dart (hunk 1 above)
Comment (EvilShadow1518) [reply lib/pages/page_22.dart:3]: ok

**Pull request closed without merging**